### PR TITLE
NMS-9215: JMS Northbounder fixes

### PR DESCRIPTION
**Pull request by dschlenk**
* JIRA: http://issues.opennms.org/browse/NMS-9215

### CODE REVIEW

**Comment by j-white** [comment]
Pushed to `pr/jira/NMS-9215`.

File: opennms-alarms/api/src/main/java/org/opennms/netmgt/alarmd/api/NorthboundAlarm.java [hunk 1]
```diff
@@ -64,7 +66,7 @@
 @XmlRootElement(name="northbound-alarm")
 @ValidateUsing("northbound-alarm.xsd")
 @XmlAccessorType(XmlAccessType.NONE)
-public class NorthboundAlarm implements Preservable {
+public class NorthboundAlarm implements Preservable, Serializable {
```
Comment (j-white) [inline opennms-alarms/api/src/main/java/org/opennms/netmgt/alarmd/api/NorthboundAlarm.java]: We should also add the`serialVersionUID` when making a class `Serializable`.

**Comment by soleger** [comment]
I'll take over the final few review tweaks here.

**Comment by soleger** [comment]
Hand-merged the local branch to foundation-2017, closing PR. Thanks @dschlenk!!

**Pull request closed without merging**